.env.example

```diff
@@ -0,0 +1,14 @@
+# If running a local setup make sure to set
+# If using docker this flag will cause it to crash
+# DB_LOCAL=true
+
+DB_USER=postgres
+DB_PASSWORD=password
+DB_NAME=chapter
+DB_URL=localhost
+DB_LOCAL=
```

Comment:
We should replace `db_local` with `is_setup_docker` and the default value should be `true` since our default setup is docker. 

Then we should add a comment which will say `set this variable to false if you are running without docker`.

Reply:
Agreed, that sounds better